### DISCUSSION TITLE
Changes for TeamTalk5

### DISCUSSION THREAD

**Discussion by DalisonCPU** [BearWare/TeamTalk5#1311] [Ideas]
I have some suggestions. I know it would be kind of boring to implement, but it would help a lot if they were heard.
1. Add unofficial servers to teamtalk 5 classic, as it may seem so, but regular teamtalk 5 is not yet ready for people with visual impairments. When you receive long messages in normal teamtalk 5, the messages are read in a strange way, and for us it is very complicated.
2. Cause servers to be ranked by popularity, that is, by the number of users on each server. From the server with the most users to the server with the least users, so it would be better to find a TeamTalk to join, not like a huge list with 100 servers with 0 users. I know a little bit of PHP, I can help with that.
3. Create an option to get the number of users on a server without the user having to connect. Thus, there would not be a bot connecting to the servers all the time. Ideally, the server would send this information to a script, but as it is, creating an option to show users when the connection is made would be nice.

**Comment by alessio0292** [comment]
hey @bear101, read this discussion that dalison written.

**Comment by bear101** [comment]
1. TeamTalk 5 Classic is no longer being developed.
2. Right-click to get context menu and select "sort by"
3. Public servers would get hundreds of logins if all clients connect to all servers in the public server list.
